sysml.library/Kernel Library/Occurrences.kerml

```diff
@@ -161,19 +167,19 @@ package Occurrences {
 	}
 	
```

Comment:
Just realized since the kerml files has an association classes for portions (TWC doesn't), they should specialize HappensDuring (aligning with the end property subsetting above):
  assoc PortionOf specialized HappensDuring.
Same for others noted below.  Can file a separate issue if you want.

Reply:
Updated. No need for a separate issue.